### ISSUE TITLE
Add "Docker Aarhus" meetup

### ISSUE THREAD

**Issue by lakruzz** [Praqma/code-conf.com#262]
Add the docker aarhus meetup http://www.meetup.com/Docker-Aarhus/events/231441979/  to the meetup pages

**Comment by lakruzz** [comment]
The issue would normally go to the "content update" milestone, which is maintained by Mona, But since the Meetup format is still being worked on by @timea and @illus0r it could be a collaboration btw @timea  and @monaslarsen 

The thing is, we want volume and diversity, to see that the format we create fits the diversity of how we actually use it.

#263 #259 #264

**Comment by timea** [comment]
@lakruzz I spoke with @monaslarsen and we decided to follow the layout of the next gen event. I showed here the includes we have for now and she liked them. This means for now we will have a summary, an agenda, a benefits and workshops include as potential lego blocks. Right now the meetups look very different on meetup.com, we thought to put an agenda on all of them (Oslo does not have one yet)

Any thoughts? Otherwise it is up to @monaslarsen to write the md files.
